README.md

```diff
@@ -1,41 +1,50 @@
 [![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)
 
-# fairlearn
+# Fairlearn
+
+The Fairlearn project seeks to enable anyone involved in the development of artificial intelligence (AI) systems to assess their system's fairness and mitigate the observed unfairness. The Fairlearn repository contains a Python package and Jupyter notebooks with the examples of usage.
+
+ #### Fairlearn components ####
```

Comment:
nit: in Markdown syntax the leading #s are sufficient. This renders like a heading of level 4. I recommend looking at the rendered output and changing to either
```
#### Fairlearn components
```
if that's what you want or
```
## Fairlearn components
```
if you want it to be a larger headline